src/rt/osx_tls.c

```diff
@@ -1,51 +0,0 @@
-/**
- * Helpers for determining TLS memory ranges on OS X.
- *
- * This unfortunately cannot be entirely done in D, as the OS X API uses
- * the Apple-specific blocks C extension.
- *
- * Copyright: David Nadlinger, 2012.
- * License:   <a href="http://www.boost.org/LICENSE_1_0.txt">Boost License 1.0</a>.
- * Authors:   David Nadlinger
- */
-
-#ifndef __APPLE__
-#ifndef __BLOCKS__
- #error "Need a C compiler with Apple Blocks support – not building on OS X with Clang?"
-#endif
-#endif
```

Comment:
So far the tests for `clang_block.d` is only enabled on OS X because it has everything required out of the box. If we install Clang and the Blocks runtime on the other machines used by the autotester I can enable it for those other platforms as well.